modules/getting-started/pages/glossary.adoc

```diff
@@ -77,23 +80,23 @@ The network allows consumers from other parts of the business to discover and us
 
 == C
 
-component::
-1. In Anypoint Platform, a POJO, Spring bean, Java bean, or a script such as Groovy, Ruby, Python, or JavaScript that contains the business logic for processing messages passing through Mule. These components typically take the whole message - or just the payload - as input. They return an object that becomes the message payload for the next element in the message processor chain.
-
-2. Generically, an architecturally specific portion of a software package.
-
-connector::
-A self-contained component that allows you to integrate Mule applications with the APIs of other, external applications, such as SalesForce, CMIS, and Twitter. A set of connectors is included in Mule version 4. Other connectors created by MuleSoft or third parties are available on Anypoint Exchange.
-
 CloudHub::
 Integration platform as a service that lets you manage applications in the cloud. When deploying an application to the cloud through the Runtime Manager console, CloudHub is used in the background.
 
 clustering::
 Using a set of Mule instances that act as, and can be managed as, a unit. A cluster is a virtual server composed of multiple nodes. You can manage cluster instances from the cloud.
 
+component::
+1. In Anypoint Platform, a POJO, Spring bean, Java bean, or a script such as Groovy, Ruby, Python, or JavaScript that contains the business logic for processing messages passing through Mule. These components typically take the whole message - or just the payload - as input. They return an object that becomes the message payload for the next element in the message processor chain.
+
+2. Generically, an architecturally specific portion of a software package.
+
 configuration builder:: 
 A class that parses a configuration file. The default configuration builder is the `org.mule.config.MuleXmlConfigurationBuilder` class, which parses a Mule XML configuration file.
 
+connector::
+A self-contained component that allows you to integrate Mule applications with the APIs of other, external applications, such as Salesforce, CMIS, and Twitter. A set of connectors is included in Mule version 4. Other connectors created by MuleSoft or third parties are available on Anypoint Exchange.
```

Comment:
I would just say Mule 4 instead of Mule version 4, but feel free to disregard.

Reply:
Agree. Fixed.